eZ/Publish/Core/Repository/PermissionsCriterionHandler.php

```diff
@@ -10,45 +10,19 @@
 
 use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
 use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LogicalAnd;
-use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LogicalOr;
-use eZ\Publish\API\Repository\Values\User\Limitation;
-use eZ\Publish\API\Repository\PermissionResolver as PermissionResolverInterface;
-use eZ\Publish\Core\Repository\Helper\LimitationService;
-use RuntimeException;
+use eZ\Publish\Core\Repository\Permission\PermissionsCriterionResolver;
 
 /**
  * Handler for permissions Criterion.
+ *
+ * @deprecated 6.12
```

Comment:
Right now this points to `6.7`. Is it ok to deprecate it there?

Reply:
Early warning? :P No you are right, either it should say 6.7.7, or this should only be applied to 6.12.